substrate/frame/verify-signature/src/benchmarking.rs

```diff
@@ -44,6 +45,8 @@ pub trait BenchmarkHelper<Signature, Signer> {
 
 #[benchmarks(where
 	T: Config + Send + Sync,
+	T::Signature: DecodeWithMemTracking,
+	T::AccountId: DecodeWithMemTracking,
```

Comment:
both will be automtically implemented if we bound it to `Parameter`.

Reply:
Answered above.

Reply:
Done